js/src/personal/jionghao/FooterSocial.tsx

```diff
@@ -0,0 +1,55 @@
+import { Container, Group, ActionIcon, rem } from '@mantine/core'
+import {
+  IconBrandTwitter,
+  IconBrandYoutube,
+  IconBrandInstagram,
+} from '@tabler/icons-react'
+import { MantineLogo } from '@mantinex/mantine-logo'
+import classes from './FooterSocial.module.css'
+
+export function FooterSocial() {
+  return (
+    <div className={classes.footer}>
+      <Container className={classes.inner}>
+        <MantineLogo size={28} />
```

Comment:
Do you actually want the Mantine logo?
If not, we likely shouldn't import the package.